### DISCUSSION TITLE
prConcurrentLimit per packageRules?

### DISCUSSION THREAD

**Discussion by fgblomqvist** [renovatebot/renovate#9460] [Request Help] [answered]
Is it possible to set the prConcurrentLimit per each packageRule?
E.g.:
* Set it to 10 for Rule 1
* Set it to 50 for Rule 2
* Set it to 30 globally

I suppose I could just test it but figured I'd ask first to see if someone knows already 😄

**Comment by viceice** [comment, marked as answer]
I don't think that's possible, because the limit will be configured before any package rule is applied
